kuzzleio/sdk-java
# Export JAVA SDK

## What does this PR do ?

Contains the C SDK's alongside cgo source files.

### How should this be manually tested?

```sh
git clone --recursive git@github.com:kuzzleio/sdk-java.git
git submodule update --init --recursive
make
cd features/java
gradle cucumber
```
You should find those files:
- build/java/build/libs/kuzzlesdk-1.0.0-javadoc.jar
- build/java/build/libs/kuzzlesdk-1.0.0-sources.jar
- build/java/build/libs/kuzzlesdk-1.0.0.jar
